torch_xla/core/xla_model.py

```diff
@@ -582,6 +585,31 @@ def all_gather(value, dim=0, groups=None, output=None, pin_layout=True):
                                            [], pin_layout)
   return result
```

Comment:
Should be removed? @jeffhataws

Reply:
yea, if you want below code to be executed. seems like otherwise it will always return here.